src/main/java/net/sf/jabref/gui/BibtexFields.java

```diff
@@ -233,6 +233,20 @@ private BibtexFields() {
         dummy.setDisplayable(false);
         add(dummy);
 
+        // IEEEtranBSTCTL fields
+        String[] ieeetranbstctlyesnofields = {"ctluse_article_number", "ctluse_paper", "ctluse_url",
```

Comment:
If we integrate this, the attributes should not be hard-coded here, but inside IEEETranEntryTypes so there is one place to administer these attributes.